components/LyricsEditor.vue

```diff
@@ -49,17 +50,6 @@ onMounted(() => {
       attributes: {
         class: "p-6 rounded-b-2xl",
       },
-      handlePaste(_, event) {
```

Comment:
that was pretty useful. I don't think we should just remove it. But maybe there's a solution that doesn't break the default copy paste. E.g. a button to remove formattings.